layers/+source-control/git/packages.el

```diff
@@ -34,6 +34,7 @@
         magit-svn
         org
         (orgit :requires org)
+        (orgit-forge :requires org)
```

Comment:
`(orgit-forge :requires (org forge))` should be sufficient to handle the dependency and then we don't need a toggle.

Reply:
Pushed a commit with the suggested change